06_modules_lab/03.py

```diff
@@ -1,11 +1,16 @@
-""" Write a program that searches current working directory
-for files larger than 1MB. Every time you find such a file print
-its name to the user.
-
-- When the program finds a large file. It should ask the user
-  a message asking if she wants to delete it, and delete the
-  file if requested
-
-- Take threshold and path as command line arguments
-"""
-
+""" 
+Delete large files (larger from 1MB by default) from specified folder (default is \) 
+"""
+#Week01_Exercise02_Assignment03
+import os
+import sys
+# Read input arguments
+root_dir, size_threshold_bytes = '\\', 1024*1024

```

Comment:
To keep the script robust, it's better to use the following to get the root path (which works for both unix AND windows):

```
os.path.abspath(os.sep)
```